Jenkinsfile

```diff
@@ -32,6 +29,7 @@ def startLisk() {
     cp test/config.json test/genesisBlock.json .
     export NODE_ENV=test
     BUILD_ID=dontKillMe ~/start_lisk.sh
+    sleep 10
```

Comment:
@MaciejBaj Jenkins runs parallel tests too fast sometimes even before Lisk node is up and running. We need them to wait a bit more until everything is settled.